src/types.ts

```diff
@@ -131,6 +134,11 @@ export interface TokenBidCollection {
   [tokenId: string]: Bid[];
 }
 
+export type TokenBidFilter =
```

Comment:
Could we make this an enum instead and then map to a string when sending to zAuction if needed?